Source/x11/XGServerEvent.m

```diff
@@ -328,6 +339,52 @@ - (NSPoint) _XPointToOSPoint: (NSPoint)x for: (void*)window
   return o;
 }
 
+- (void) initializeMouse
+{
+  [self mouseOptionsChanged:nil];
+  [[NSDistributedNotificationCenter defaultCenter]
+    addObserver: self
+       selector: @selector(mouseOptionsChanged:)
+           name: NSUserDefaultsDidChangeNotification
+         object: nil];
+}
+
+- (void) mouseOptionsChanged: (NSNotification *)aNotif
+{
+  NSUserDefaults *defs = [NSUserDefaults standardUserDefaults];
+
+  clickTime = [defs integerForKey:@"GSDoubleClickTime"];
+  if (clickTime < 250)
```

Comment:
Shouldn't we use a lower number as the limit here, e.g. 100, and have the old value 300 as default?

Reply:
From the UX point of view 100 is too low number. The human reaction time is 270-280 milliseconds on average (look here for example: https://www.humanbenchmark.com/tests/reactiontime/statistics). That's why 250 ms. I've tried to set 100 ms and I must say that this is too little time between clicks. Practically we shouldn't allow to set such low number to prevent user annoyance.

Reply:
If 100 is to low, then some other sensible value like 200, but the important bit is to keep the default of 300. So if somebody won't set this value she will get the previous behaviour.

Reply:
OK, changed.